### PR TITLE
Fix login user

### PR DESCRIPTION
**Pull request by shpandrak**
Using the default login admin
Adding documentation for default user

### CODE REVIEW

File: deployer/deployer.go [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ func postCommandOrcs(
 	}
 
 	req.Header.Add("Content-Type", "application/json")
-	req.SetBasicAuth("shpandrak", "1234")
+	req.SetBasicAuth("admin", "nazgul")
```
Comment (vjtirumalai) [inline deployer/deployer.go]: Wouldn't you be better off defining these constants once and use those in all places?

Comment (shpandrak) [reply deployer/deployer.go]: Yes it will, fixing

**Comment by shpandrak** [comment]
ok, comments fixed. please verify

**Comment by shpandrak** [comment]
@vjtirumalai, in order to test the changes you'll have to build the orcs docker image locally from the branch that includes the fix...

**Pull request merged**